### PR TITLE
Fixing failing tests

### PR DESCRIPTION
**Pull request by Blaizzy**
- PTL requirements
- Tensorflow requirements
- how-to-guides/how-it-works/scripts
- how-to-guides/monitor-ml-runs/scripts

### CODE REVIEW

**Comment by Blaizzy** [comment]
@kamil-kaczmarek no action is needed at the moment here. This is still in progress, I'm just running the tests to analyze them and see if my tests take any effect.

**Comment by Blaizzy** [comment]
@kamil-kaczmarek, all tests are passing!

You can review it.

Notes: 

I fixed all issues. Most were about downgrading the versions of the libraries because I noticed that some errors said that the specified higher version of the library doesn't exist while it exists.

Example:
`ERROR: No matching distribution found for tensorflow==2.7.2`

Proof of that exists in Pypi: https://pypi.org/project/tensorflow/2.7.2/

I did some research and saw that it could be related to the Python version of the machine. In our case, we are using v3.6, while TF 2.7.2 might require v3.7+.

https://github.com/tensorflow/tensorflow/issues/53709#issuecomment-1015816806

Finally, I also found out that the **dependabot** is the culprit of most of the issues, as it updates the dependencies from tested versions to higher ones without considering Python version support.
Example:
PR https://github.com/neptune-ai/examples/pull/91

**Comment by kamil-kaczmarek** [comment]
@Blaizzy 
I dropped support for Python 3.6, trying to revert TF to 2.7.1

File: how-to-guides/how-it-works/scripts/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 neptune-client
 scikit-plot==0.3.7
-tensorflow==2.7.2
\ No newline at end of file
+tensorflow==2.4.1
```
Comment (kamil-kaczmarek) [inline how-to-guides/how-it-works/scripts/requirements.txt:3]:
```suggestion
tensorflow==2.7.2
```

File: how-to-guides/monitor-ml-runs/scripts/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,2 @@
 neptune-client
-tensorflow==2.7.2
\ No newline at end of file
+tensorflow==2.4.1
```
Comment (kamil-kaczmarek) [inline how-to-guides/monitor-ml-runs/scripts/requirements.txt:2]:
```suggestion
tensorflow==2.7.2
```

File: integrations-and-supported-tools/tensorflow-keras/scripts/requirements.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
 neptune-client
 neptune-tensorflow-keras
-tensorflow==2.7.2
\ No newline at end of file
+tensorflow==2.4.1
```
Comment (kamil-kaczmarek) [inline integrations-and-supported-tools/tensorflow-keras/scripts/requirements.txt:3]:
```suggestion
tensorflow==2.7.2
```

**Review by kamil-kaczmarek — CHANGES_REQUESTED** [review]

**Review by kamil-kaczmarek — APPROVED** [review]

**Pull request merged**